### DISCUSSION TITLE
is there a way to show only the next event in a calendar

### DISCUSSION THREAD

**Discussion by Nemesis24** [totaldebug/atomic-calendar-revive#1302] [Support]
1rst tx for all 
 that's a really good job

Is there a way to show only the next event like an anniversary 

sorry for my bad english
tx for your answers.

**Comment by marksie1988** [comment]
you can limit the calendar by x number of events:

```
maxEventCount: 1
```

**Comment by Nemesis24** [comment]
ok fine thank you i will try it tomerrow
